radis/io/hitran.py

```diff
@@ -1160,10 +1174,31 @@ def download_all_hitran_isotopes(molecule, directory):
                         )
                         os.remove(join(directory, file + ".data"))
                 try:
-                    fetch(file, get_molecule_identifier(molecule), iso, wmin, wmax)
-                except KeyError:
-                    # Isotope not defined:
-                    continue
+                    if extra_params == "all":
+                        fetch(
+                            file,
+                            get_molecule_identifier(molecule),
+                            iso,
+                            wmin,
+                            wmax,
+                            ParameterGroups=[*PARAMETER_GROUPS_HITRAN],
+                        )
+                    elif extra_params is None:
+                        fetch(file, get_molecule_identifier(molecule), iso, wmin, wmax)
+                    else:
+                        raise ValueError("extra_params can only be 'all' or None ")
+                except KeyError as err:
+                    list_pattern = ["(", ",", ")"]
+                    import re
+
+                    if (
+                        set(list_pattern).issubset(set(str(err)))
+                        and len(re.findall("\d", str(err))) >= 2
```

Comment:
Thats not a very explicit error? Do you know what's the typical error message when a isotope is not found?

Reply:
I tried printing the err and the typical message that comes is  => ("molecule_id","isotope_id") . for instance-> `(5,7)` then `(5,8)` and then `(5,9)` in the loop in case of CO.

Reply:
Ok . Then add this sentence in the docstrings (the regex are cryptic)  .
You can also collect the 2 integers; and check that the first one corresponds to the expected molecule_id
This we're making sure it's an isotope Key error and not something else

use `get_molecule_identifier`: 
```python
from radis.db.classes import get_molecule_identifier 

# ex : 
# >>> get_molecule_identifier ("CO")  
# 5
```

Reply:
Yes, have done the changes in the recent commit